x-pack/test/functional/apps/monitoring/elasticsearch/index_detail.js

```diff
@@ -38,7 +38,7 @@ export default function ({ getService, getPageObjects }) {
         await tearDown();
       });
 
-      it('green status index with full shard allocation', async () => {
+      it('should have an index summary with green status index with full shard allocation', async () => {
```

Comment:
nit: Tests that specifically write `should` in their title seem redundant. It's a test, so the fact the behavior you're testing should happen is implied. To save some characters, you could leave off the "should" and this would be `it('has an index summary with...`

Reply:
I started adding `should` in the first part of the description because it is super helpful for knowing what to grep for when running the test, and what to search for in the code. I used to not do it, then I realized how useful it is